ios/Classes/BetterPlayer.m

```diff
@@ -764,20 +785,27 @@ - (void)pictureInPictureControllerDidStartPictureInPicture:(AVPictureInPictureCo
 
 - (void)pictureInPictureControllerWillStopPictureInPicture:(AVPictureInPictureController *)pictureInPictureController  API_AVAILABLE(ios(9.0)){
     [self setIsPipMode:false];
+    [self hideBlackCoverView];
+    [self hideLimitedPlanCoverView];
     
     bool wasPlaying = _isPlaying;
     if (_eventSink != nil) {
         _eventSink(@{@"event" : @"exitingPIP",
                      @"wasPlaying" : @(wasPlaying),
                    });
+        if (!_isPlaying && !_isLiveStream) {
+            _eventSink(@{@"event" : @"seek",
```

Comment:
Sorry.
Do you really want to do the whole "seek" event?

How about handling it when receiving the `exitingPIP` event on the app-side?

Reply:
> Do you really want to do the whole "seek" event?

Absolutely yes, this event only trigger if the player is still pausing.

As i mention in the PR's description, when we return to normal mode, the app side still keep the old position (the position before entering PIP), then the progress bar, the playback time will display wrong, and more serious than that is the "handle premium banner" function which depend on that wrong position could make the premium section got leaked.

Reply:
> How about handling it when receiving the exitingPIP event on the app-side?

Do you mean we should put `position` into  `_eventSink(@{@"event" : @"exitingPIP")` event and call the `seekTo` in the app side then?

Reply:
@mvn-hienvu-hn 
Thank you for describing.
I understand the current situation.

> Do you mean we should put position into _eventSink(@{@"event" : @"exitingPIP") event and call the seekTo in the app side then?

Yes.
You can do that? 🙏 
(I think, it is difficult to describe that reason on betterplayer library's code. And we may need to separate the original "seekTo" event from this event in the future.)

Reply:
@noboru-i 

I made those changes here ([21c1b6d](https://github.com/dwango-nfc/betterplayer/pull/54/commits/21c1b6d8505f75ab500b911485d2dd3e5d6e5e22), [7a6cab1](https://github.com/dwango-nfc/dwango-app-ch/pull/1811/commits/7a6cab11fac6946f8a9ea67476283a448ccbe9f0))
Could you please check it 🙇?